src/components/ChatLog.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry.js'
+
+const ChatLog = (props) => {
+    if (!props.chatData) {
+        return null
+    }
+    const ChatLogEntries = props.chatData.map((entry) => {
+        return (
+            <div key={entry.id}>
+                <ChatEntry 
+                    id={entry.id} 
+                    sender={entry.sender} 
+                    timeStamp={entry.timeStamp} 
+                    body={entry.body} 
+                    liked={entry.liked}
+                    onUpdateChat={props.onUpdateChat}
+                    chatEntry={entry.sender===props.chatData[0].sender ? 'chat-entry local' : 'chat-entry remote'}
+                ></ChatEntry>
+            </div>
+        );
+    });
```

Comment:
Small note that variables don't begin with a capital letter. The prop key should go on each ChatEntry, not on the entire div. When you have an array of components (like you have an array of ChatEntries), each component in the array gets a prop key.

Also, instead of writing a method `ChatLogEntries` to reference the return value from calling map() on `props.chatData`, it's more common to directly iterate over the chat entries in your component like this without using a variable: 
```javascript
const ChatLog = (props) => {
    return (
     <div>
        {props.entries.map((message) => (
            <ChatEntry 
                key={entry.id} // the key should go on each ChatEntry
                id={entry.id}
                sender={entry.sender}
                body={entry.body}
                timeStamp={entry.timeStamp}
                liked={entry.liked}
                onUpdateChat={props.onUpdateChat}
            />
        ))}
    </div>
    )
};
``` 

The logic on line 20 to determine the class name for remote or local should live in ChatEntry.js I think you can omit line 20 completely and have this logic live in the child component. I think this because each child component should know who its sender is and thus should be in charge of determining the class.